nextmv/nextmv/cloud/application.py

```diff
@@ -796,6 +858,36 @@ def list_instances(self) -> list[Instance]:
 
         return [Instance.from_dict(instance) for instance in response.json()]
 
+    def list_ensemble_definitions(self) -> list[EnsembleDefinition]:
+        """
+        List all ensemble_definitions.
+
+        Returns
+        -------
+        list['EnsembleDefinition']
```

Comment:
```suggestion
        list[EnsembleDefinition]
```

I think in this case backticks are not needed.